packages/harmony/.babelrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
just checking this was created automatically by storybook init yeah?

Reply:
no, does storybook init normally create the babel config?

Reply:
not sure, just wondering why we needed the .babelrc in the first place. i guess it's what harmony has basically?